src/functions/json-validator.js

```diff
@@ -45,6 +45,7 @@ export async function jsonValidator(exclude) {
   const yamlExtension = core.getInput('yaml_extension')
   const yamlExtensionShort = core.getInput('yaml_extension_short')
   const useDotMatch = core.getBooleanInput('use_dot_match')
+  let files = core.getInput('files').split('\n').filter(Boolean)
```

Comment:
```suggestion
  let files = core.getMultilineInput('files')
```

I think you could leverage `core.getMultilineInput()` here.

Example source in the `actions/checkout` project: https://github.com/actions/checkout/blob/96f53100ba2a5449eb71d2e6604bbcd94b9449b5/src/input-helper.ts#L86

Then you could do something like this for end users:

<img width="351" alt="Screenshot 2023-08-11 at 12 13 01 PM" src="https://github.com/GrantBirki/json-yaml-validate/assets/23362539/107b458f-527e-4cde-907e-ea6be3695c6c">

---

[source code](https://github.com/actions/toolkit/blob/45c49b09df04cff84c5f336f07d5232fa7103761/packages/core/src/core.ts#L140-L161)

Reply:
I have not used the `core.getMultilineInput()` before but I think that should help you out here

Reply:
Done